autocomplete/builders/common.lua

```diff
@@ -393,54 +408,102 @@ end
 -- Package compilation
 --
 
+---@alias packageType
+---|"class"
+---|"function"
+---|"method"
+---|"value"
+---|"lib"
+---|"event"
+
 --- @class exampleTable
 --- @field title string|nil The example title.
 --- @field description string|nil The description of the example.
 
 --- @class package
+--- @field name string
+--- @field useDefault boolean
```

Comment:
Are you sure the package can have `useDefault` field?

Reply:
😂  I'll remove that. I was a bit confused when I started this PR.